### PR TITLE
page: Added an about us page

### PR DESCRIPTION
**Pull request by dolapojohnson**
This PR adds an about us link to the Header and the about us page to the pages folder.

### CODE REVIEW

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]
@dolapojohnson Thankyou very much. Here are some suggestions. Also Please make the page UI consistent with other tabs for better visibility.

image

File: public/assets/group.png
[the remark this answers is not in the document]
Comment (dolapojohnson) [reply public/assets/group.png:1]: Image was downloaded from freepik

**Comment by dolapojohnson** [comment]
Please kindly review and let me know if it requires any fixes. Thank you

**Review by Harshil-Jani — APPROVED** [review]

**Comment by Harshil-Jani** [comment]
Thanks @dolapojohnson

**Pull request merged**